protocol/driver.go

```diff
@@ -85,6 +85,13 @@ func AsDriver(cc *grpc.ClientConn) driver.Driver {
 	}
 }
 
+func AsDriverFromClient(c DriverClient, h DriverHostClient) driver.Driver {
```

Comment:
`DriverFromClient` would be good enough name IMHO

Reply:
Done